Mathlib/Geometry/Manifold/Sheaf/LocallyRingedSpace.lean

```diff
@@ -85,12 +85,14 @@ theorem smoothSheafCommRing.isUnit_stalk_iff {x : M}
         ext y
         apply mul_inv_cancel
         exact hVf y
+        · infer_instance
```

Comment:
Is this asking for a `FunLike` here?

Reply:
No. The `convert map_one _` line above generates a `OneHomClass` goal. This can be avoided with `RingHom.map_one`

Reply:
Same below